go.mod

```diff
@@ -11,21 +11,25 @@ require (
 	github.com/drone/envsubst/v2 v2.0.0-20210305151453-490366e43a3c
 	github.com/evanphx/json-patch v4.11.0+incompatible
 	github.com/fatih/color v1.12.0
+	github.com/form3tech-oss/jwt-go v3.2.3+incompatible // indirect
 	github.com/go-logr/logr v0.4.0
 	github.com/gobuffalo/flect v0.2.2
+	github.com/golang/groupcache v0.0.0-20210331224755-41bb18bfe9da // indirect
 	github.com/google/go-cmp v0.5.6
 	github.com/google/go-github/v33 v33.0.0
 	github.com/google/gofuzz v1.2.0
 	github.com/gosuri/uitable v0.0.4
+	github.com/mattn/go-runewidth v0.0.9 // indirect
 	github.com/onsi/ginkgo v1.16.4
 	github.com/onsi/gomega v1.13.0
 	github.com/pkg/errors v0.9.1
 	github.com/spf13/cobra v1.1.3
 	github.com/spf13/pflag v1.0.5
 	github.com/spf13/viper v1.7.1
-	go.etcd.io/etcd v0.5.0-alpha.5.0.20200910180754-dd1b699fc489
```

Comment:
Is the RC better or worse than shipping with this `alpha` tag? 😂

Reply:
It's not really alpha because the etcd repo doesn't have tags compatible with go mod. It's between v3.4.13 and v3.4.14 for a specific commit that's also imported in k/k.

Reply:
Ah ok, we should have a comment on that. It always catches me off guard 😄

Reply:
https://github.com/kubernetes-sigs/cluster-api/pull/4770